engine/src/main/java/org/terasology/rendering/nui/widgets/UIButton.java

```diff
@@ -36,26 +36,48 @@
 import java.util.List;
 
 /**
+ * A widget displaying a clickable button, containing text and an optional image
  */
 public class UIButton extends CoreWidget {
     public static final String DOWN_MODE = "down";
 
+    /**
+     * The {@link Binding} containing the {@link TextureRegion} corresponding to the image shown on the button
+     */
     @LayoutConfig
     private Binding<TextureRegion> image = new DefaultBinding<>();
 
+    /**
+     * The {@code Binding} containing the text to be shown on the button
+     */
     @LayoutConfig
     private Binding<String> text = new DefaultBinding<>("");
 
+    /**
+     * The {@code Binding} containing the {@link StaticSound} to be played when the button is clicked
+     */
     @LayoutConfig
     private Binding<StaticSound> clickSound = new DefaultBinding<>(Assets.getSound("engine:click").get());
 
+    /**
+     * The {@code Binding} containing the float representing the volume of the click sound, 1.0 by default
```

Comment:
This is the concrete OpenAL call that applies the volume to a sound: https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/audio/openAL/OpenALManager.java#L191. AFAICT there aren't any restrictions, so it can be an arbitrary multiplier.